validators/filterResidentialBuildings/README.md

```diff
@@ -1,6 +1,20 @@
 #### Description
 
-Filter buildings with tag `building=residential`. Returns a line delimeted feature collection to stdout.
+Filter buildings with tag `building=residential`. Returns a line delimeted feature collection to stdout. The linter will return a JSON bin for edit recency statistics with the following format:
+
+```
+{
+  'YYYYM': number of buildings,
+  '20138': 75,
+  '20139': 416,
+  '20140': 363,
```

Comment:
@geohacker is the month zero indexed? on month 10,11,12 shouldn't the format be "YYYYMM" ?

Reply:
You're right. Moment uses zero indexed months. I'd say we keep it this way. https://momentjs.com/docs/#/get-set/month/